### ISSUE TITLE
Transformation Type

### ISSUE THREAD

**Issue by thehabes** [thehabes/GeoreferencePrototype#2]
Our prototype is missing this information.  What are some ways we can supply this information?

**Comment by thehabes** [comment]
@bert where might this go?  If the type specifically applies to "`pixel_coords`", it may be worth making "`pixel_coords`" a JSON object like 

```JSON
pixel_coords:{
   "type": "TransformationType",
   "value":[12, 12]
}
```

**Comment by thehabes** [comment]
If the type is put on the GeoJSON body, we would have to careful because GeoJSON already uses `type`.  `type` can be a @set.  So..

```JSON
{
  "type": ["Feature", "TransformationType"],
  "properties": {
    "pixel_coords": [
      2840,
      3153
    ]
  },
  "geometry": {
    "type": "Point",
    "coordinates": [
      -78.137998,
      39.217154
    ]
  }
}
```
or extend GeoJSON with something like `transformation`.

```JSON
{
  "type": "Feature",
  "transformation":"Type",
  "properties": {
    "pixel_coords": [
      2840,
      3153
    ]
  },
  "geometry": {
    "type": "Point",
    "coordinates": [
      -78.137998,
      39.217154
    ]
  }
}
```

I am more in favor of keeping our stuff as scoped as possible, so I would prefer going with something like in the previous comment where this new field/type are defined alongside the "`pixel_coords`".  The less extending we have to do, the better.

**Comment by bertspaan** [comment]
What about something like:

```json
{
  "body": {
    "type": "FeatureCollection",
    "transformationType": "polynomial",
    "features": [
    ]
  }
}
```

**Comment by thehabes** [comment]
Yes, that would work out. `transformationType` would be considered a foreign member to the GeoJSON.  We could add a context entry for transformationType and scope in some expected values.  We would then add all of that to the vocabulary document.

Is it always true that all Features will use the same transformationType as noted by the FeatureCollection?  
Or, another way to ask: Is there a situation where you would need different transformationTypes for Features in the same FeatureCollection?  If so, we may need to accommodate that immediately and put transformationType on the Feature.  

Here are the rules for foreign members : https://tools.ietf.org/html/rfc7946#section-6.1

I believe what you and I are suggesting follows all the rules.
